modules/op-setting-annotations-labels-namespace.adoc

```diff
@@ -0,0 +1,28 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-setting-annotations-labels-namespace_{context}"]
+= Setting labels and annotations for the {pipelines-shortname} installation namespace
+
+You can set labels and annotations on the `openshift-pipelines` namespace in which the operator installs {pipelines-shortname}.
+
+[NOTE]
+====
+Changing the name of the `openshift-pipelines` namespace is not supported at present
```

Comment:
```suggestion
Changing the name of the `openshift-pipelines` namespace is not supported.
```
Alternatively, you can replace "at present" with "Currently, changing the name of the...." . But I prefer the suggested change.